com.unity.render-pipelines.high-definition/Documentation~/HDR-Output.md

```diff
@@ -0,0 +1,96 @@
+# High Definition Range (HDR) Output in HDRP
+
+With HDRP it is possible to output to High Dynamic Range (HDR) devices that allows for wider color gamut and higher brightness range.
+
+HDR Output can be enabled for a given platform in **Player Settings > Other Settings > Use display in HDR mode**.  When this option is enabled, HDRP will process the frame so that when an HDR screen is used as output device it is presented correctly.
+
```

Comment:
Proposed rewrite:

"[High Dynamic Range](https://en.wikipedia.org/wiki/High-dynamic-range_imaging) content has a wider color gamut and greater luminosity range than standard definition content.  

HDRP can output HDRP content for devices which support that functionality. 

## Enabling HDR Output

To enable HDR output, navigate to **Project Settings > **Player** > **Other Settings** and enable **Use display in HDR mode**.

Reply:
All good, did make the change